test/test_click_api_consistency.py

```diff
@@ -17,19 +17,25 @@ def is_click_command(obj: Any) -> bool:
 class ClickApiConsistency(unittest.TestCase):
 
     @staticmethod
-    def _replace_list_with_tuple(x: Any):
+    def _adjust_default_value_for_multiple(x: Any):
         """
         Click stores default values as list if 'Multiple'=true. However, for plain Python methods we
         need to use (immutable) tuples for declaring default value. Hence, we need to convert lists with tuples
         for comparison.
+        Also, default values for none-required parameters of with multiple=true are marked as "None"
+        in the command object, but in reality click invokes the command with "tuple()" for that parameter.
+        (Bug in click????)
```

Comment:
If you are not sure, write check that project's bug tracker. Write a ticket please, if the project has no related ticket.

Reply:
On one hand I agree with you that we should report bugs. But on the other hand I the test uses some internal structures from click, which is not what normal clients are supposed to do. I don't believe it's worth the effort to open, track and respond the ticket.